### ISSUE TITLE
java jdk install warnings

### ISSUE THREAD

**Issue by paguilera** [saltstack-formulas/sun-java-formula#26]
Saw this one when provisioning this morning. In the `unpack-jdk-tarball` section.

```
              ----------
                        ID: unpack-jdk-tarball
                  Function: archive.extracted
                      Name: /usr/share/java
                    Result: True
                   Comment: file:///usr/share/java/jdk-7u80-linux-x64.tar.gz extracted in /usr/share/java/
                   Started: 13:41:06.197343
                  Duration: 8217.83 ms
                   Changes:   
                            ----------
                            directories_created:
                                - /usr/share/java/
                                - /usr/share/java/jdk1.7.0_80
                            extracted_files:
                                - jdk1.7.0_80
                                - jdk1.7.0_80/LICENSE
                                - jdk1.7.0_80/release
                                - jdk1.7.0_80/jre
                                - jdk1.7.0_80/jre/LICENSE
                                - jdk1.7.0_80/jre/Welcome.html
                                - jdk1.7.0_80/jre/plugin

                                <SNIP>
                  Warnings: 'group', 'options' and 'user' are invalid keyword arguments for
                            'archive.extracted'. If you were trying to pass additional data to
                            be used in a template context, please populate 'context' with
                            'key: value' pairs. Your approach will work until Salt Carbon is
                            out. Please update your state files.
```

**Comment by blbradley** [comment]
I think I'm seeing something similar in elasticsearch-formula's test runs over the past day.

**Comment by blbradley** [comment]
Closed in #28. @paguilera Feel free to reopen if you feel the issue has not be resolved.
